addons/product/models/product_template.py

```diff
@@ -662,6 +658,14 @@ def _create_variant_ids(self):
         self.invalidate_cache()
         return True
 
+    @staticmethod
+    def generate_variant_dict(tmpl_id, combination):
```

Comment:
a static class with a record as first param :thinking:

Reply:
This was resolved with the latest changes (can't mark it as resolved as I'm not the owner of the PR)